### PR TITLE
Feature/change map type

### PR DESCRIPTION
**Pull request by pgilliar**
add a button to change the map between the normal type and the satellite type

### CODE REVIEW

File: app/src/main/res/layout/activity_maps.xml [hunk 1]
```diff
@@ -32,13 +32,29 @@
         app:srcCompat="@android:drawable/ic_dialog_map"
         tools:ignore="SpeakableTextPresentCheck" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```
Comment (TheblueMan003) [inline app/src/main/res/layout/activity_maps.xml:35]: The button is displayed on top of the navigation bar
image

Comment (pgilliar) [reply app/src/main/res/layout/activity_maps.xml:35]: it should be nice now

**Review by TheblueMan003 — APPROVED** [review]
Looks good otherwise

**Review by atappy — APPROVED** [review]
nice, good job, enhanced the experience greatly

**Pull request merged**